tests/main.sh

```diff
@@ -25,13 +27,27 @@ import_subdir_files() {
 import_subdir_files includes
 
 # If adding a test suite, then ensure to add it here to be picked up!
-TEST_NAMES="test_static_analysis \
-            test_branches \
-            test_cli \
-            test_controller \
-            test_cmr_bundles \
-            test_machine \
-            test_smoke"
+TEST_NAMES="static_analysis \
+            appdata \
```

Comment:
appdata and model do not exist in 2.7 as suites.

Reply:
I need to back port more, to bring in better checking!

Reply:
Is there a bug, or card for this?  Or remove them now?

Reply:
ah, yeah, good point